boards/DCM/Src/Io/Io_IsrHalCallback.c

```diff
@@ -0,0 +1,14 @@
+#include <stm32f3xx_hal.h>
+#include "main.h"
+#include "Io_LSM6DS33.h"
```

Comment:
```suggestion
```

Reply:
actually wait no, the compiler needs this include to recognize the Io_LSM6DS33_ExtiCallback() function